### DISCUSSION TITLE
Thanks very much for this App

### DISCUSSION THREAD

**Discussion by pctechnikch** [heliomarpm/udemy-downloader-gui#41] [General]
Finally, the application is now working correctly again. The login is now working again. And skipping files works again. That's great and that's about all that needs to be said.

Greetings René

**Comment by heliomarpm** [comment]
Thanks, I'll keep trying to make you better every day

**Comment by jagatkishoremishra** [comment]
Where to down load the application .Please help me with the URL

**Reply by heliomarpm** [reply]
https://github.com/heliomarpm/udemy-downloader-gui/releases/latest

**Reply by espaol** [reply]
It doesn't work anymore, I have tried a lot to login with gmail but it doesn't anymore. Anyone faced something like that?

**Reply by heliomarpm** [reply]
@espaol login with google has really been down for a while, I'll look at this problem again soon to try to solve it.
Login with username and password still working perfectly

**Reply by heliomarpm** [reply]
Another alternative is to connect with your token
